src/Orchard.Environment.Shell.Abstractions/Builders/ShellContext.cs

```diff
@@ -27,7 +27,7 @@ public class ShellContext : IDisposable
         /// </summary>
         public IServiceScope CreateServiceScope()
         {
-            return ServiceProvider.GetRequiredService<IServiceScopeFactory>().CreateScope();
+            return ServiceProvider.CreateScope();
         }
```

Comment:
Oh, yes you're right, we could remove it, or keep it as an helper just to write `shellContext.CreateScope()` in place of `shellContext.ServiceProvider.CreateScope()`.